src/main/java/com/minecolonies/coremod/colony/ColonyList.java

```diff
@@ -15,47 +15,27 @@
 import java.util.stream.StreamSupport;
 
 //TODO extend list/collection
+
 /**
  * Data structure for storing colonies, optimized for performance.
  *
- * @author Colton
- *
  * @param <T> Type of IColony (Colony or ColonyView)
+ * @author Colton
  */
 public final class ColonyList<T extends IColony> implements Iterable<T>
 {
     @VisibleForTesting
-    static final int INITIAL_SIZE = 16;
-
-    private IColony[] list = new IColony[INITIAL_SIZE];
-
-    private final List<Integer> nullIndices = new ArrayList<>();
-
-    private int topID = 0;
+    static final  int           INITIAL_SIZE = 16;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Explicitly declare the visibility for "INITIAL_SIZE". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2039)